### ISSUE TITLE
Bug in Phony usage

### ISSUE THREAD

**Issue by ezzatron** [philiplb/CRUDlex#59]
Hi there, I noticed you have a bug in the way you're using Phony. Lines like [this one](https://github.com/philiplb/CRUDlex/blob/ad5a7b935defe182a321ffc31c8726ace0c0ccaa/tests/CRUDlexTests/ControllerProviderTest.php#L100):

```php
$this->fileProcessorHandle->updateFile->never();
```

Should probably be:

```php
$this->fileProcessorHandle->updateFile->never()->called();
```

As calling `never()` by itself does nothing. See [Verifying that a spy event happened an exact number of times](http://eloquent-software.com/phony/0.13/#verifying-that-a-spy-event-happened-an-exact-number-of-times).

**Comment by philiplb** [comment]
Now that's what I call service, thank you for noting. :)
Fixed in 272c55c

**Comment by ezzatron** [comment]
No problem, it's fun to see people really using it in the wild :)
